api/src/pcapi/core/mails/transactional/sendinblue_template_ids.py

```diff
@@ -32,4 +33,4 @@ class TransactionalEmail(Enum):
 @dataclasses.dataclass
 class SendinblueTransactionalEmailData:
     template: Template
-    params: dict
+    params: typing.Optional[dict] = dataclasses.field(default=None)
```

Comment:
(détail) est-ce que...

```suggestion
    params: typing.Optional[dict] = None
```

... ne fonctionne pas ?

Reply:
D'un autre côté, est-ce que la valeur par défaut ne devrait pas être un dictionnaire vide, plutôt que `None` ? Ça ne changerait rien pour le code qui l'utilise, mais ça me semblerait plus évident. Et le lecteur ne se poserait pas la question : "quelle différence y a-t-il entre des params à `None` et des params valant `{}` ?"

```suggestion
    params: dict = dataclasses.field(default_factory=dict)
```

Reply:
Yes ça marche, ça évite aussi les `Optional[dict]`